metaswitch/common/mib.py

```diff
@@ -59,6 +62,8 @@ def get_oids_at_depth(self, depth):
 class Statistic(object):
     ''' The class structure for each OID and its relevant information
     '''
+    def __repr__(self):
+        return self.details['SNMP NAME']

```

Comment:
It's a good rule of thumb that you should be able to re-create an object with `eval(repr(x))`. That way you know you've captured everything about the object.